partnerships/how-to-guides/github-issues.md

```diff
@@ -0,0 +1,37 @@
+# How do I open a GitHub issue with Engineering?
+
+This page describes how you can open a general GitHub issue with the {team}`Engineering Team`, for example, to report a failing CI workflow, ask for a feature enhancement or even deploy a new Hub. In this guide, we will be using the {doc}`../../projects/managed-hubs/showcase-hub` as our example GitHub repo.
+
+## Steps
+
+1. Navigate to the Community Showcase [GitHub repo](https://github.com/2i2c-org/community-showcase).
+2. Under the Issues tab, click **New issue**.
+
+```{image} ../media/github-issues/01-new-issue.png
+:alt: Screenshot of the GitHub issues webpage, highlighting the New issue button.
+```
+
+3. If applicable, select the most appropriate GitHub  (or open a blank issue). In this example, we use the 💡 **General Issue** template.
+
+4. Fill out the template
+
+    - Populate the text input fields. Try to be concise while providing as much technical detail as you can. For complicated issues, providing a Minimal Working Example and relevant error messages from output logs where possible is a great help. Use code blocks and avoid pasting screenshots of code.
+    - If you know the 2i2c engineer that is responsible for responding to this issue use the **Assignees** field on the right-hand side to assign it to them so that they can add this issue to their Sprint Board. Otherwise, post a comment in the issue tagging the GitHub team **@2i2c-org/engineering** to ping the entire team.
```

Comment:
I think the other thing that could happen here is that the issue gets added to the [Engineering Backlog project board](https://github.com/orgs/2i2c-org/projects/22/views/25) and @damianavila is pinged so that he knows to consider it when shaping the next sprint. I don't think we should encourage the habit of bypassing our engineering management processes too often as it could end up with an engineer trying to complete more work than they have capacity for.

Reply:
I thought that was likely the case! Updated in https://github.com/2i2c-org/team-compass/pull/792/commits/3f674962087dcc243c68ece69a8e2eedefbf5d12 to assign to @damianavila to triage to the appropriate engineer when capacity is available.